godot-core/src/builtin/string/node_path.rs

```diff
@@ -49,6 +80,50 @@ impl NodePath {
             .expect("Godot hashes are uint32_t")
     }
 
+    /// Returns the property name indicated by `idx`, starting from 0. If `idx` is out of bounds, [`None`] is returned.
+    /// See also [`NodePath::get_subname_count`].
+    pub fn get_subname(&self, idx: u32) -> Option<StringName> {
```

Comment:
See above for potential docs.

Reply:
```rust
    /// Returns the property name at position `idx`.
    ///
    /// If `idx` >= [`get_subname_count()`][Self::get_subname_count], `None` is returned.
    ///
    /// If you want to get a node name instead, check out [`get_name()`][Self::get_name].
    /// 
    /// # Example
    /// ```no_run
    /// # use godot::prelude::*;
    /// let path = NodePath::from("../RigidBody2D/Sprite2D:rotation:y");
    /// godot_print!("{}", path.get_subname(0).unwrap()); // "rotation"
    /// godot_print!("{}", path.get_subname(1).unwrap()); // "y"
    /// ```
    pub fn get_subname(&self, idx: u32) -> Option<StringName> {
```
Does this sound acceptable?

Reply:
Yes, great! :+1: 

(regarding `.unwrap`, see main thread)